locator/util.cc

```diff
@@ -0,0 +1,144 @@
+#include "locator/util.hh"
```

Comment:
Copyright and licensing comment missing. You can copy-n-paste one from pretty much any other .cc file

Reply:
added